File: twosum/twosum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = []
        index = 1
        for i in nums:
     
            for j in nums[index:]:
                if i+j == target:
                    if i == j:
                        result = [n for n, x in enumerate(nums) if x == i]
                    else:
                        result.append(nums.index(i))
                        result.append(nums.index(j))
                    return result
            index += 1

File: twosum/twosum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let dict = {}
    for (let i = 0; i < nums.length; i++) {
        complement = target - nums[i]
        if(complement in dict){
            return [dict[complement], i]
        }
        dict[nums[i]] = i
    }
};
